ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -721,6 +721,10 @@ private int executeIncrementalLoad(long loadStartTime) throws Exception {
     Database targetDb = getHive().getDatabase(work.dbNameToLoadIn);
     Map<String, String> props = new HashMap<>();
 
+    if(targetDb == null) {
```

Comment:
Please follow the coding convention; have a whitespace after if.

Reply:
Done.